kf_lib_data_ingest/app/cli.py

```diff
@@ -18,6 +19,25 @@
 
 CONTEXT_SETTINGS = {"help_option_names": ["-h", "--help"]}
 DEFAULT_LOG_LEVEL_NAME = logging._levelToName.get(DEFAULT_LOG_LEVEL)
+DEFAULT_VALIDATION_MODE = VALIDATION_MODES[1]
+VALIDATION_MODE_OPT = {
+    "args": ("--validation_mode",),
+    "kwargs": {
+        "default": DEFAULT_VALIDATION_MODE,
+        "type": click.Choice(VALIDATION_MODES),
+        "help": (
+            "Does not apply if --no_validate CLI flag is present. "
+            "The `basic` mode runs validation faster but may not find 100% of "
+            "the errors if the data contains implied connections between "
+            "entity types (e.g. You have a file that relates participants to "
+            "source genomic files and a file that relates participants to "
+            "biospecimens, instead of a file which relates participants "
+            "directly to biospecimens). The `advanced` mode runs validation "
+            "slower but is guarenteed to find all errors even if your data "
+            "has implied connections."
```

Comment:
This description isn't right. We can't say whether advanced mode will flag more things or fewer, just that it will be more thorough. The relationships discovered by that thoroughness may be good or bad.

Reply:
Ok, can you give me a better description?